app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -0,0 +1,26 @@
+class DradisDatatable {
+  constructor(selector) {
+    this.$table = $(selector);
+    this.dataTable = null;
+    this.init();
+  }
+
+  init() {
+    var dataTable = this.$table.DataTable({
```

Comment:
```suggestion
    this.dataTable = this.$table.DataTable({
```

or do we need the local variable here?

Reply:
You're right, we don't need it here. thanks